src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -85,8 +84,8 @@ export interface SpecimenDefinition {
   slug: string;
   derived_from_uri: string | null;
   status: Status;
-  description: string | null;
-  type_collected: Code | null;
+  description: string;
+  type_collected: Code;
   patient_preparation: Code[] | null;
   collection: Code | null;
```

Comment:
```suggestion
  collection?: Code | null;
```

<img width="831" alt="image" src="https://github.com/user-attachments/assets/1c095294-02ea-479f-96fe-3d79d214f97a" />